tap.js

```diff
@@ -28,8 +29,29 @@
         this.el.addEventListener('mousedown', this, false);
     }
 
-    Tap.prototype.start = function(e) {
+    // query which button was pressed in the event
+    // handles many browsers/versions
+    // returns: 1=left, 2=right, 4=center, 8=4th, 16=5th
+    Tap.prototype.whichbutton = function(event) {
+        // modern & preferred:  MSIE>=9, Firefox(all)
+        if ('buttons' in event) {
+           // http://www.w3schools.com/jsref/event_buttons.asp
```

Comment:
Please don't rely on or reference articles from w3schools. They are both horribly inaccurate as well as a poor source of reference for other developers. I'd suggest using MDN instead.